app/services/abstract_search_service.rb

```diff
@@ -113,7 +113,11 @@ def search(request_or_query)
             @query_url.to_s % { q: URI.escape(request_or_query.to_s), max: Settings.MAX_RESULTS }
           end
 
-    response = Faraday.get url.to_s
+    response = Faraday.get do |req|
+      req.url url.to_s
+      # Passing a known "non-Stanford" ipaddress (in the IBM range)
+      req.headers['X-Forwarded-For'] = '9.9.9.9'
```

Comment:
can we use a private IP number like 192.168.0.1?

Reply:
Sure thing @drh-stanford .. updated to use that IP address.